src/validation.js

```diff
@@ -20,6 +20,10 @@ export default class AuroFormValidation {
     const name = tagName.slice(tagNameDivider + 1);
     const camelCaseName = nameSpace + name.charAt(0).toUpperCase() + name.slice(1);
 
+    if (elem.tagName.toLowerCase().startsWith('auro-input')) {
+      return 'auroInput';
+    }
```

Comment:
All of these inputs need to be changed to use a generic function that supports any component that you pass a tagname in as a string and then you look to see if the element actually uses that tagname or if (due to custom registration) has the tagname as an attribute.